NEWS.md

```diff
@@ -1,5 +1,11 @@
 # datawizard 1.2.0
```

Comment:
Can you fix the version number in NEWS?

There's also some spelling and styling to fix and then it's good to go.

Reply:
Added `1.2.0.xx`

Reply:
Thanks, just styling and spelling left ;)